packages/mui-material/src/Paper/Paper.js

```diff
@@ -39,7 +39,7 @@ const PaperRoot = styled('div', {
       ownerState.variant === 'elevation' && styles[`elevation${ownerState.elevation}`],
     ];
   },
-})(({ theme }) => ({
+})(styled.pure((theme) => ({
```

Comment:
Not entirely sure how PigmentCSS works and if this would mess up its extraction logic though.